ts/src/handlers.ts

```diff
@@ -548,6 +559,7 @@ export class Handlers {
                 signedTransaction,
                 coordinatorOrders,
                 takerAssetFillAmounts,
+                Handlers._isWhitelistedTakerContract(txOrigin) ? txOrigin : undefined,
```

Comment:
We want to check if the `takerAddress` is whitelisted here. We're dealing with a single taker contract that we know in advance, but we have no way of knowing if `txOrigin` should be whitelisted.